x11-themes/numix-gtk-theme/numix-gtk-theme-9999.ebuild

```diff
@@ -1,33 +1,38 @@
 # Copyright 1999-2015 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=6
 
-MY_PN="Numix"
-DESCRIPTION="the Numix flat theme for gtk."
-HOMEPAGE="https://github.com/shimmerproject/${MY_PN}"
+inherit eutils
 
-if [[ ${PV} == "9999" ]] ; then
-	inherit git-r3
-	SRC_URI=""
-	EGIT_REPO_URI="https://github.com/shimmerproject/${MY_PN}.git"
-	KEYWORDS=""
+DESCRIPTION="A modern flat theme with a combination of light and dark elements. It supports GNOME, Unity, Xfce and Openbox."
+HOMEPAGE="https://numixproject.org"
+
+BASE_URI="https://github.com/numixproject/${PN}"
+
+if [[ ${PV} == *9999 ]]; then
+    inherit git-r3
+    SRC_URI=""
+    EGIT_REPO_URI="${BASE_URI}.git"
+    KEYWORDS=""
 else
-	SRC_URI="https://github.com/shimmerproject/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
-	KEYWORDS="~amd64 ~arm ~x86"
+    SRC_URI="${BASE_URI}/archive/${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Spaces here and also I think you didn't test this for a non 9999 version of the script. Indeed removing the v you are no longer able to download the source file.